### PR TITLE
tags and collections

### PR DESCRIPTION
**Pull request by chris-luangrath**

### CODE REVIEW

File: footron_controller/controller.py [hunk 1]
```diff
@@ -51,17 +55,53 @@ def __init__(self):
     def load_from_fs(self):
         self.load_experiences()
         self.load_collections()
+        self.load_tags()
+        self.fill_collection_dictionary()
```
Comment (chris-luangrath) [inline footron_controller/controller.py]: Do we want these to be separate functions?

File: footron_controller/data/tags.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+from dataclasses import dataclass
+from typing import List, Optional
+
+from ..constants import BASE_DATA_PATH
+
+
+@dataclass
+class Tag:
+    id: str
+    experiences: Optional[list]
```
Comment (vinhowe) [inline footron_controller/data/tags.py]: `List` has to be capitalized. You already have an import so you should be good.

File: footron_controller/data/collection.py [hunk 3]
```diff
@@ -23,7 +22,7 @@ def load_collections_from_fs(path=BASE_DATA_PATH) -> List[Collection]:
     with open(collections_file_path) as collections_file:
         collection_data = json.load(collections_file)
 
-    for item in collection_data:
-        collections.append(Collection(**item))
-
+    for key, value in collection_data.items():
+        collections.append(Collection(id=key, **value))
+
```
Comment (vinhowe) [inline footron_controller/data/collection.py]: Remove this indent.

File: footron_controller/data/collection.py [hunk 4]
```diff
@@ -8,9 +9,7 @@
 @dataclass
 class Collection:
     id: str
-    title: str
-    description: Optional[str] = None
-    artist: Optional[str] = None
+    experiences: list
```
Comment (vinhowe) [inline footron_controller/data/collection.py]: Same as below, should be `List`.

**Review by vinhowe — CHANGES_REQUESTED** [review]

File: footron_controller/data/loader.py [hunk 5]
```diff
@@ -6,6 +6,7 @@
 
 from ..util import mercilessly_kill_process
 from ..constants import BASE_BIN_PATH
+
```
Comment (vinhowe) [inline footron_controller/data/loader.py]:
```suggestion
```

**Review by vinhowe — APPROVED** [review]
LGTM

**Pull request merged**